tests/core/test_optimizations.py

```
@@ -0,0 +1,153 @@
+"""
+Unit tests for core optimizations module.
+Tests cache management and performance tracking functionality.
+"""
+import pytest
+import asyncio
+import time
```

Ruff (0.8.2)

`time` imported but unused

Remove unused import: `time`

(F401)